persistent/tests/test_persistence.py

```diff
@@ -1627,6 +1675,10 @@ def update_object_size_estimation(self, oid, new_size):
 
         return _Cache(jar)
 
+    def _makeRealCache(self, jar):
+        from persistent.picklecache import PickleCache
```

Comment:
CPersistentTests already used cPickleCache in _makeCache, not a mock like the PyPersistentTests did.